sycl/test-e2e/Plugin/level_zero_ext_intel_cslice.cpp

```diff
@@ -53,11 +53,20 @@ bool IsPVC(device &d) {
   return masked_device_id == 0xbd0 || masked_device_id == 0xb60;
 }
 
+bool IsPVC_MultiTiles(device &d) {
+  // PVC-1T (one tile) does not support partitioning by affinity domain,
+  // which this test requires
+  if (!isPartitionableByAffinityDomain(d))
+    return false;
+
+  return IsPVC(d);
+}
+
 void test_pvc(device &d) {
   std::cout << "Test PVC Begin" << std::endl;
   // CHECK-PVC: Test PVC Begin
-  std::cout << "IsPVC: " << IsPVC(d) << std::endl;
-  if (IsPVC(d)) {
+  std::cout << "IsPVC: " << IsPVC_MultiTiles(d) << std::endl;
+  if (IsPVC_MultiTiles(d)) {
```

Comment:
I think we should adjust the test to check that even 1T cards can be partitioned by CSlice.